app/views/core_induction_programmes/modules/show.html.erb

```diff
@@ -26,12 +26,26 @@
         <li class="app-task-list__item">
           <h2 class="govuk-heading-m">
             <% if current_user.admin? %>
-              <%= govuk_link_to lesson.title, lesson_path(lesson) %>
+              <%= govuk_link_to lesson.title_for(current_user), lesson_path(lesson) %>
             <% else %>
-              <%= lesson.title %>
+              <%= lesson.title_for(current_user) %>
             <% end %>
           </h2>
-          <p class="govuk-body"><%= lesson.ect_summary %></p>
+
+          <% if current_user&.admin? %>
+            <% if lesson.ect_summary %>
+              <h3 class="govuk-heading-s">ECT summary:</h3>
+              <p class="govuk-body"><%= lesson.ect_summary.presence || "No summary available" %></p>
+            <% end %>
+            <% if lesson.mentor_summary %>
+              <h3 class="govuk-heading-s">Mentor summary:</h3>
+              <p class="govuk-body"><%= lesson.mentor_summary.presence || "No summary available" %></p>
+            <% end %>
+          <% elsif current_user.mentor? %>
+            <p class="govuk-body"><%= lesson.mentor_summary.presence || "No summary available" %></p>
+          <% else %>
+            <p class="govuk-body"><%= lesson.ect_summary.presence || "No summary available" %></p>
+          <% end %>
```

Comment:
Hmm, I think summaries should behave like titles - when there is no mentor summary, always show ect summary. (we do have ect summaries for modules everywhere, but no mentor summaries for them, and, ehh, who knows how soon we will get that sorted).

Reply:
Feel free to make a separate PR for it if you want :)

Reply:
I'm using the behaviour specified on the ticket, which looked pretty specific: https://dfedigital.atlassian.net/browse/CPDEL-472

Reply:
> GIVEN the user is a mentor
THEN display the mentor-specific weekly description on the weekly breakdown screen

Nothing about having a `No summary available`. If anything, we should display absolutely nothing if the summary is missing. Could you double check with Kam?

Reply:
kam says leave blank! i'll change it